assets/queries/cloudFormation/ec2_subnet_unable_connect_cluster/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [result]  {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::EC2::Subnet"
+  map := resource.Properties.MapPublicIpOnLaunch
+  checkState(map)
+  result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties.MapPublicIpOnLaunch", [name]),
+                "issueType":		"IncorrectValue",  
+                "keyExpectedValue": sprintf("'Resources.%s.Properties.MapPublicIpOnLaunch' should be false",[name]),
+                "keyActualValue": 	sprintf("'Resources.%s.Properties.MapPublicIpOnLaunch' is true",[name])
+              }
+}
+checkState(map) = true {
+	map == true
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline